include/boost/algorithm/cxx11/copy_if.hpp

```diff
@@ -126,6 +126,82 @@ copy_until ( const Range &r, OutputIterator result, Predicate p )
     return boost::algorithm::copy_until (boost::begin (r), boost::end(r), result, p);
 }
 
+/// \fn copy_if_while ( InputIterator first, InputIterator last, OutputIterator result, CopyPredicate copy_pred, TerminatePred term_pred )
+/// \brief Copies all the elements from the input range that satisfy the
```

Comment:
You missed something here; shouldn't this be:
Copies all the elements from the input range that satisfy the predicate while the termination predicate is satisfied.

also, is `TerminatePred` the correct term to use here?

Reply:
I fixed the missing documentation here.

Reply:
I'm not sure if `TerminatePred` is correct - I definitely struggled with naming of the predicates to indicate what they were and how they were different - these were the best that I could come up with. If you have a better name, I am more than happy to make changes!